policies/templates/gcp_vpc_sc_ensure_access_levels_v1.yaml

```diff
@@ -64,16 +74,38 @@ spec:
              	lib.has_field(asset, "service_perimeter")
              
              	lib.get_constraint_params(constraint, params)
-             	required_access_levels_array := lib.get_default(params, "required_access_levels", [])
-             	required_access_levels := {p | p = required_access_levels_array[_]}
+             	mode := lib.get_default(params, "mode", "require")
```

Comment:
should the default be require or required?

Reply:
It should be `require`. I made a typo in the enum which is now fixed.